build_win32.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -eu
+
+BRAVE_TOR_VERSION="1"
+TOR_VERSION="0.3.2.10"
+
+rm -rf Tor/
+rm -rf Data/
+
+curl -fsSL "https://dist.torproject.org/torbrowser/7.5.4/tor-win32-0.3.2.10.zip" -o tor-win32-0.3.2.10.zip
+curl -fsSL "https://dist.torproject.org/torbrowser/7.5.4/tor-win32-0.3.2.10.zip.asc" -o tor-win32-0.3.2.10.zip.asc
+
+gpg --keyserver pgp.mit.edu/ --recv D1483FA6C3C07136
```

Comment:
also needs the long key ID. https://github.com/brave/tor_build_scripts/pull/1/files#r187763561